### PR TITLE
Improved audio-driven mouth shapes

### PR DESCRIPTION
**Pull request by PhilipRosedale**
Changes the algorithm used to drive the mouth movement from audio only.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2377875623f5c35e0df12dd0579d6049cf8fab0e available at: https://gist.github.com/fe9e13e076d7429658fc<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6496.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6496.dmg

**Comment by imgntn** [comment]
@PhilipRosedale not sure exactly what i should be looking for in testing this, but most of the avatars I tried (Mery, Lovejoy, Owen) look pretty good.  Albert has some weird upper lip flaring that goes on.  Functionally, things seem generally to work.

File: interface/src/avatar/Head.cpp [hunk 1]
```diff
@@ -246,28 +248,32 @@ void Head::calculateMouthShapes() {
...
+    const float FUNNEL_SPEED = 2.335f;
+    const float STOP_GAIN = 5.0f;
+
+    // From the change in loudness, decide how much to open or close the jaw
     float audioDelta = sqrtf(glm::max(_averageLoudness - _longTermAverageLoudness, 0.0f)) * JAW_OPEN_SCALE;
     if (audioDelta > _audioJawOpen) {
         _audioJawOpen += (audioDelta - _audioJawOpen) * JAW_OPEN_RATE;
-    } else {
+    }
+    else {
```
Comment (hyperlogic) [inline interface/src/avatar/Head.cpp]: Coding convention. The closing and opening brace should be on the same line.

    } else {

**Comment by hifi-gustavo** [comment]
Build console output for commit: be2f14cae580d675b1cea1c499a256f38aa91cc4 available at: https://gist.github.com/fe9e13e076d7429658fc<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6496.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6496.dmg

**Comment by hyperlogic** [comment]
:+1:

**Pull request merged**